js/best-practices.md

```diff
@@ -1,5 +1,58 @@
 # Javascript / Best practices
 
+#### Avoid exporting an object for the purpose of exporting its properties
+
+[Axel Rauschmayer](https://medium.com/@rauschma/note-that-default-exporting-objects-is-usually-an-anti-pattern-if-you-want-to-export-the-cf674423ac38):
+> Note that default-exporting objects is usually an anti-pattern (if you want to export the properties). You lose some ES6 module benefits ([tree-shaking](https://webpack.js.org/guides/tree-shaking/) and [faster access to imports](https://2ality.com/2014/09/es6-modules-final.html#benefit-1%3A-faster-lookup)).
+
+Instead, prefer breaking up the code into separate default exported modules or use named exports. See also:
+- https://geedew.com/es6-module-gotchas/
+- https://github.com/airbnb/javascript/issues/710#issuecomment-297840604
+- https://2ality.com/2014/09/es6-modules-final.html#default-exports-are-favored
+
+BAD
+
+```js
+export default { a, b, c };
+```
```

Comment:
Could you add a "GOOD" example here?

Reply:
https://github.com/c-hive/guides/pull/38/commits/170d514168f22a5c8a58d898ec7f925f4952bb1b

Reply:
Thanks!